### ISSUE TITLE
Add TheStoryLord to staff list

### ISSUE THREAD

**Issue by ExcaliburZero** [Infinite-Dango-Studios/ganbatte#27]
@TheStoryLord has just joined the studio as a helper. He is going to be able to help out with various different things.

Currently he is not on any of the staff lists, so he will need to be added to them.

@TheStoryLord
For your first task you should add your username to the staff list in the README file for Ganbatte. In order to do this you are going to:
- Go to the Ganbatte repository: https://github.com/Infinite-Dango-Studios/ganbatte
- Click on the button in the top right of the screen that says "Fork" in order to create a copy of the repository to your own account.
- Go to your fork of the Ganbatte repository.
- Click on the README.md file in the file list of your forked repository.
- Click on the pencil icon that is towards the top-right of the file on the screen.
- Find the Staff section of the file.
- Copy and paste one of the staff member lines and edit it to use your username, the url of your GitHub profile page, and your title "Helper".
- Click on "Preview changes" towards the top left of the page to make sure the new entry on the staff list is working properly. That it uses your username, links to your profile page, and uses your title.
- Once it is ready, scroll to the bottom of the page where it says "Commit changes".
- Enter the following into the following into the first smaller box that by default says "Update README.md":
  - Added TheStoryLord to the staff list
- Then click on the radio bubble next to "Create a new branch for this commit and start a pull request.".
- Click on the button that says "Propose file change".
- The website will then take you to the pull request creation page, but you will not want to make a pull request here.
- Instead, go back to the main Ganbatte repository: https://github.com/Infinite-Dango-Studios/ganbatte
- Click on the "Pull requests" section on the right side of the page.
- Click on the green "New pull request" button.
- Click on the blue text toward the top right of the page that says "compare across forks".
- Click on the grey button that says "head fork" and select your fork of the repository.
- Click on the grey button that says compare to the right of the "head fork" button. Select the branch you created (not the master branch).
- Click on the green button that says "Create pull request".
- Fill in some info on the pull request in the forms that pop up.
- Then click on the green button that says "Create pull request".

If you need any help doing that, just let me know.

**Comment by ExcaliburZero** [comment]
TheStoryLord has been added to the game's main README with #28.

**Comment by ExcaliburZero** [comment]
TheStoryLord needs to be added to:
- [x] Game end credits
- [x] Game credits screen
- [x] Website staff page
- [x] Website press kit

**Comment by ExcaliburZero** [comment]
Added TheStoryLord to the game credits screen and end credits with 08192b7f3deac99e1a9bb0cfa96389742e83bcad .

**Comment by ExcaliburZero** [comment]
Added TheStoryLord to the website staff page and press kit.

Since all the staff lists have bee updated, I will now close this issue.
